src/blog/2022/11/flowforge-1-1-released.md

```diff
@@ -1,36 +1,67 @@
 ---
 title: FlowForge 1.1 released
-subtitle: Node-RED editor on Devices, persistant file storage and import snapshots from outside FlowForge
-description: Node-RED editor on Devices, persistant file storage and import snapshots from outside FlowForge
+subtitle: In FlowForge 1.1, we are continuing to add the features we believe makes FlowForge the best solution for running Node-RED in production in a secure, scalable, and team-based environment.
```

Comment:
<img width="903" alt="Screenshot 2022-11-24 at 08 10 18" src="https://user-images.githubusercontent.com/2529595/203717034-504f3a28-a43e-4a49-9e01-2dc0f30073b6.png">

The way this renders creates a stutter. 

```suggestion
subtitle: A new set of features to improve FlowForge as the best solution for running Node-RED in production in a secure, scalable, and team-based environment.
```